arch/riscv/include/uapi/asm/kvm.h

```diff
@@ -149,6 +149,7 @@ enum KVM_RISCV_SBI_EXT_ID {
 	KVM_RISCV_SBI_EXT_VENDOR,
 	KVM_RISCV_SBI_EXT_DBCN,
 	KVM_RISCV_SBI_EXT_COVG,
+	KVM_RISCV_SBI_EXT_COVH,
```

Comment:
nit: Separate patch for any uapi changes.

Reply:
done. I moved it to commit #240aefdd272002d09443a2822431904afa538fba